libs/common-components/src/lib/modal/modal.component.less

```diff
@@ -64,7 +75,7 @@
   cursor: pointer;
   font-size: 12px;
 }
-@media (max-width: 1025px) {
+@media (max-width: 1037px) {
```

Comment:
use this variables https://github.com/groupdocs-total/GroupDocs.Total-Angular/blob/efd282e3f186a025605632d226c578c115c4a54d/libs/common-components/src/styles/variables.less#L1-L5 instead of exact values